server/views/pages/exhibition.njk

```diff
@@ -50,6 +52,14 @@
           </div>
           <hr class="divider divider--keyline divider--pumice {{ {s:2} | spacingClasses({margin: ['bottom']}) }}" />
           {% block accessStatements %}
+            <div class="flex {{ {s:1} | spacingClasses({margin: ['top', 'bottom']}) }} {{ {s:'HNL4'} | fontClasses}}">
```

Comment:
I hadn't realised that this stuff will always be true (makes sense now).

Perhaps the _linked_ access statements will need to be moved into e.g. their own section further down the page (from memory, they're a fair bit longer and I think will quickly look quite awkward in this space).